be/src/vec/exec/vanalytic_eval_node.cpp

```diff
@@ -571,7 +571,16 @@ Status VAnalyticEvalNode::_output_current_block(Block* block) {
     }
 
     for (size_t i = 0; i < _result_window_columns.size(); ++i) {
-        block->insert({std::move(_result_window_columns[i]), _agg_functions[i]->data_type(), ""});
+        SlotDescriptor* output_slot_desc = _output_tuple_desc->slots()[i];
```

Comment:
I think we should fix this bug in another way. Specifically, I think we should figure out why `_agg_functions[i]->data_type()` here is not nullable. Maybe we should change it to nullable in planning phase